dash/visualisations.py

```diff
@@ -30,6 +30,6 @@ def create_visualisation(df: pd.DataFrame, x: str, y:str) -> px.line:
 		max_hr = sql.get_list("SELECT max_hr FROM user_info")[0][0]
 		x_values = df.duration.tolist()
 		y_values = [max_hr]*len(x_values)
-		fig.add_trace(go.Scatter(x=x_values, y=y_values, mode='lines', line_color="red")) # Add line for max heart rate
+		fig.add_trace(go.Scatter(x=x_values, y=y_values, mode='lines', line_color="red", name='Max')) # Add line for max heart rate
 	return fig
```

Comment:
Nice addition to the heart rate graph. Still having problems with every user being 220 bpm or is this fixed now?

Reply:
No problems now, it seemed to fix itself, suspect I was working with an older version of the file in my branch, which updated after a pull.